docs/contract/logs.md

```diff
@@ -22,15 +22,13 @@ Version: 0.0
 - **ID field** - _optional_
   - The first string field with the name `id` is the id field.
   - Unique identifier of the log line
-- **Attributes field** - _optional_
-  - The first string field with the name `attributes` is the attributes field.
-  - Other systems may refer to this with other names, for example Loki calls these "labels"
-  - This field represent additional attributes of the log line.
+- **Labels field** - _optional_
+  - The first field with the name `labels` is the labels field.
+  - This field represent additional labels of the log line.
   - Field type must be json raw message type. Example value: `{}`, `{"hello":"world", "foo": 123.45, "bar" :["yellow","red"], "baz" : { "name": "alice" }}`
-    - Should not be empty string.
```

Comment:
I might be missing something, but `labels` is not a string field, so I don't understand this point and therefore I removed it. Let me know if I just misunderstood something and we should keep this.